File: uwuifier/uwufier.py
import random

uwutext = input("pwease input text uwu: ")
# replacing letters uwu
# TODO add a percentage for stuttering
uwutext = (
    uwutext.lower()
    .replace("l", "w")
    .replace("r", "w")
    .replace("v", "f")
    .replace("i", "i-i")
    .replace("d", "d-d")
    .replace("n", "n-n")
    + " >~<"
)


print(uwutext)

File: uwuifier/uwufier.cpp
#include <algorithm>
#include <string>
#include <iostream>

using namespace std;

void replaceString()
{
    string stutter = "i";
    string s;
    cout << "pwease entew text uwu: " << endl;
    getline(cin, s);
    replace(s.begin(), s.end(), 'l', 'w');
    replace(s.begin(), s.end(), 'r', 'w');
    replace(s.begin(), s.end(), 'v', 'f');

    cout << s + " >~<" << endl;
}

int main()
{

    replaceString();
    return 0;
}
